components/header/_header.scss

```diff
@@ -1,30 +1,65 @@
-$header-bp: 36em;
-
+// Block
 .header {
   display: block;
-  overflow: hidden;
-  padding: 1.4rem 0 1rem;
-  &::before {
-    content: "";
-    display: block;
+  position: relative;
+  margin: 0 0 2rem;
+
+  // Elements
+  &__top {
     width: 100%;
-    height: 0.45rem;
-    background-color: $color-secondary;
-    position: absolute;
-    top: 0;
-    left: 0;
+    padding: .7rem 0;
+    background: $color-secondary;
+  }
+
+  &__logo {
+    height: 1.333rem;
+  }
+
+  &__main {
+    padding: 1.1rem 0;
+    background: $color-header-main;
+  }
+
+  &__nav {
+    background: $color-header-main;
+    @include mq(xs,s) {
+      background: darken($color-header-main, 5%);
+    }
+  }
+
+  &__title {
+    color: $color-white;
+    font-size: 1.556rem;
+    margin: 0;
+  }
+
+  &__desc {
+    color: $color-white;
     margin: 0;
   }
-  @include mq($header-bp) {
-    display: flex;
-    align-items: center;
+
+  // Modifier - Hero variant
+  &--hero {
+    .header {
+      &__title {
+        font-size: 2rem;
+      }    
+    }
   }
-}
 
-.header__logo {
-  margin-bottom: 1rem;
-  @include mq($header-bp) {
-    margin-bottom: 0;
-    flex: 1 1 100%;
+  // Modifier - Thin variant
+  &--thin {
```

Comment:
Do we need a thin version? Might cause a bit of confusion with different services.

Reply:
@bennpearson This was created for internal facing services. The read me explains it.